src/internal/types.ts

```diff
@@ -45,7 +45,11 @@ export interface Subscribable<T> {
 }
 
 export type ObservableInput<T> = SubscribableOrPromise<T> | ArrayLike<T> | Iterable<T>;
-export type ObservableLike<T> = { [Symbol.observable]: () => Subscribable<T>; };
+
+/** @deprecated use {@link InteropObservable } */
+export type ObservableLike<T> = InteropObservable<T>;
+
+export type InteropObservable<T> = { [Symbol.observable]: () => Subscribable<T>; };
```

Comment:
Just curious, but why the new name?

Reply:
@alfaproject I guess it's because `Observable` is not assignable to `ObservableLike`. Every other type with a `Like` suffix that I can think of is assignable to its namesake type that does not have the suffix.